### PR TITLE
Support Java 11 (7.x)

### PR DESCRIPTION
**Pull request by zoliMolnar**
This pull request adds minimalistic support for Java 11.

### CODE REVIEW

File: releng/target-platform/.launch/snow-owl-oss.launch [hunk 1]
```diff
@@ -22,8 +22,13 @@
 <stringAttribute key="pde.version" value="3.3"/>
 <stringAttribute key="product" value="com.b2international.snowowl.server.product.product"/>
 <stringAttribute key="productFile" value="\com.b2international.snowowl.server.update\so_server.product"/>
+<<<<<<< HEAD
```
Comment (apeteri) [inline releng/target-platform/.launch/snow-owl-oss.launch]: ⚠️ Merge conflict markers are still present in this file.

**Review by apeteri — CHANGES_REQUESTED** [review]

**Comment by apeteri** [comment]
I also have some questions:

- Did any of the test projects break due to compile errors? I saw this with newer Eclipse releases where the test classpath is separate from regular classes.
- Does classgraph support PDE devMode classpaths now?

**Comment by apeteri** [comment]
In the Travis build, Tycho did not like `JavaSE-11` as an execution environment value: https://travis-ci.org/b2ihealthcare/snow-owl/builds/469543966

**Comment by zoliMolnar** [comment]
@apeteri Neither of the test projects broke due to compilation errors. 
ClassGraph supports it but currently it has a bug with the newer versions of eclipse (specifically versions from 2018-09). To resolve this I've forked ClassGraph, fixed the bug and opened a PR for it.

**Comment by cmark** [comment]
Please bump Tycho to v1.3 to be able to build projects with Java 11.
https://wiki.eclipse.org/Tycho/Release_Notes/1.3

**Pull request merged**